doc/domains.rst

```diff
@@ -1103,6 +1103,22 @@ The JavaScript Domain
 
 The JavaScript domain (name **js**) provides the following directives:
 
+.. rst:directive:: .. js:package:: name
```

Comment:
Sorry, I don't know the keyword "package" in JavaScript language.
Did you mean the package of node.js?
I think "module" is better. It is added since ES6.